imports/server/hunts.ts

```diff
@@ -37,10 +37,10 @@ const existingJoinEmail = (user: Meteor.User | null, hunt: HuntType, joinerName:
 };
 
 Meteor.methods({
-  addToHunt(huntId: string, email: string) {
+  addToHunt(huntId: unknown, email: unknown) {
     check(huntId, String);
     check(email, String);
-    if (!this.userId) throw new Meteor.Error(401, 'Unauthorized');
```

Comment:
You sure that `this.userId` is always either a full user ID or undefined, and not empty string?  Just want to be sure we're not dropping a necessary authentication check.

Reply:
Yeah, the [Meteor docs](https://docs.meteor.com/api/methods.html#DDPCommon-MethodInvocation-userId) document the semantics of this field. I added a handful of these checks [when I converted to TypeScript](https://github.com/deathandmayhem/jolly-roger/commit/f1e73440b93c187b240d726dfb38eedda6bec192#diff-f4c92cab507de9189ac164cf7c595bb6R43) so that the typechecker would see the value as non-null later on.